dask/dataframe/io/json.py

```diff
@@ -170,13 +170,15 @@ def read_json(url_path, orient='records', lines=None, storage_options=None,
 def read_json_chunk(chunk, encoding, errors, kwargs, meta=None):
     s = io.StringIO(chunk.decode(encoding, errors))
     s.seek(0)
-    df = pd.read_json(s, orient='records', lines=True, **kwargs)
+    read_function = kwargs.pop('read_function', pd.read_json)
+    df = read_function(s, orient='records', lines=True, **kwargs)
     if meta is not None and df.empty:
         return meta
     else:
         return df
 
 
 def read_json_file(f, orient, lines, kwargs):
+    read_function = kwargs.pop('read_function', pd.read_json)
```

Comment:
We might want to document the `read_function` parameter in the `dd.read_json` docstring. Otherwise it may not be obvious it exists without looking at the source code.